### DISCUSSION TITLE
MP4 file indicating not natively supported

### DISCUSSION THREAD

**Discussion by HiTechKoke** [mifi/lossless-cut#704] [Support]
Have MP4s from Skydio 2 Drone and when I drag them into the tool I get the dialog "File not natively supported Preview may have no audio or low quality...".  Using latest lossless version33.33.1 on a MAC / Catalina 10.15.4.  When I try to play the clip it is really blurry.  Don't think this is normal for MP4s from what I am reading they are supported.  Also tried to convert to to supported format but get another dialog saying "File not natively supported..." when using the Fast conversion options.  Seemed to work on one of the slow options I tried.   Is there something unique about this MP4 file?

**Comment by mifi** [comment]
What is the codec? Could u please provide a report (help menu)

**Reply by HiTechKoke** [reply]
Just filed the Help ->Error report.  Did not realize that was a feature.

**Reply by mifi** [reply]
Oh, I dont believe I received any report

**Reply by HiTechKoke** [reply]
Sorry... didn't realize I needed to paste it here.  I'll read better next time.

No error

{
  "state": {
    "filePath": "/Users/kevinsmith/DocumentsLocal/Skydio-Berryessa-3-13-21/S1000098.MP4",
    "fileFormat": "mp4",
    "externalStreamFiles": [],
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "hevc",
        "codec_long_name": "H.265 / HEVC (High Efficiency Video Coding)",
        "profile": "Main",
        "codec_type": "video",
        "codec_time_base": "16666667/500000000",
        "codec_tag_string": "hvc1",
        "codec_tag": "0x31637668",
        "width": 3840,
        "height": 2160,
        "coded_width": 3840,
        "coded_height": 2176,
        "closed_captions": 0,
        "has_b_frames": 0,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "16:9",
        "pix_fmt": "yuvj420p",
        "level": 183,
        "color_range": "pc",
        "color_space": "bt709",
        "color_transfer": "bt709",
        "color_primaries": "bt709",
        "refs": 1,
        "r_frame_rate": "30/1",
        "avg_frame_rate": "204120000/6802753",
        "time_base": "1/90000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 13605506,
        "duration": "151.172289",
        "bit_rate": "100017632",
        "nb_frames": "4536",
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0
        },
        "tags": {
          "creation_time": "2021-03-13T22:06:21.000000Z",
          "language": "eng",
          "handler_name": "VideoHandle"
        }
      },
      {
        "index": 1,
        "codec_name": "mjpeg",
        "codec_long_name": "Motion JPEG",
        "profile": "Baseline",
        "codec_type": "video",
        "codec_time_base": "0/1",
        "codec_tag_string": "[0][0][0][0]",
        "codec_tag": "0x0000",
        "width": 384,
        "height": 216,
        "coded_width": 384,
        "coded_height": 216,
        "closed_captions": 0,
        "has_b_frames": 0,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "16:9",
        "pix_fmt": "yuvj420p",
        "level": -99,
        "color_range": "pc",
        "color_space": "bt470bg",
        "chroma_location": "center",
        "refs": 1,
        "r_frame_rate": "90000/1",
        "avg_frame_rate": "0/0",
        "time_base": "1/90000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 13605840,
        "duration": "151.176000",
        "bits_per_raw_sample": "8",
        "disposition": {
          "default": 0,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 1,
          "timed_thumbnails": 0
        }
      }
    ],
    "copyStreamIdsByFile": {
      "/Users/kevinsmith/DocumentsLocal/Skydio-Berryessa-3-13-21/S1000098.MP4": {
        "0": true,
        "1": false
      }
    },
    "cutSegments": [
      {
        "start": 13.105635,
        "end": 19.096499456202235
      },
      {
        "start": 29.061339486576525,
        "end": 31.560881300215563
      }
    ],
    "fileFormatData": {
      "filename": "/Users/kevinsmith/DocumentsLocal/Skydio-Berryessa-3-13-21/S1000098.MP4",
      "nb_streams": 2,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "151.176000",
      "size": "1892786333",
      "bit_rate": "100163323",
      "probe_score": 100,
      "tags": {
        "major_brand": "mp42",
        "minor_version": "0",
        "compatible_brands": "isommp42",
        "creation_time": "2021-03-13T22:06:21.000000Z",
        "com.android.version": "8.1.0",
        "com.android.manufacturer": "Skydio",
        "com.android.model": "2",
        "com.android.capture.fps": "30.000000",
        "location": "+38.5815-122.2369/",
        "location-eng": "+38.5815-122.2369/"
      }
    },
    "rotation": 360,
    "shortestFlag": false
  },
  "platform": "darwin",
  "version": "3.33.1"
}

**Reply by mifi** [reply]
Oh, I see that your video is h265, which cannot be natively supported in losslesscut yet unfortunately. there is an issue for that, see #88

**Comment by HiTechKoke** [comment]
Ok tks for the info... maybe a more specific message relating to H.265 in
the warning message would be helpful to call it out since it is very
popular now on many platforms, including Apple.  In the end I can still do
rough cuts to allow me a better way to reduce down my videos to take them
to the next stage.  Tks for the responses and making this all available.

Kevin
